### PR TITLE
📝(project) add a specific CHANGELOG.md for the helm chart

### PR DESCRIPTION
**Pull request by wilbrdt**
## Purpose

Changes on Ralph Helm chart are decorrelated from Ralph changes.

## Proposal

Creating a separated helm chart CHANGELOG to keep track of these changes.

### CODE REVIEW

File: src/helm/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+All notable changes to this helm chart will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to
+[Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
+
+## [0.2.0] - 2023-11-08
```
Comment (jmaupetit) [inline src/helm/CHANGELOG.md:12]: Do you think you can manually add tags to the repository pointing to those changes, and add related link at the end of this changelog?

Comment (wilbrdt) [reply src/helm/CHANGELOG.md:12]: Done!

**Review by quitterie-lcs — APPROVED** [review]
LGTM

File: src/helm/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Changelog
+
+All notable changes to this helm chart will be documented in this file.
```
Comment (Leobouloc) [inline src/helm/CHANGELOG.md:3]:
```suggestion
All notable changes to the helm chart will be documented in this file.
```

**Review by Leobouloc — APPROVED** [review]
Approved !

**Pull request merged**